underdocs-version/src/main/kotlin/underdocs/version/reader/PropertiesReader.kt

```diff
@@ -0,0 +1,7 @@
+package underdocs.version.reader
+
+import java.util.Properties
+
+interface PropertiesReader {
+    fun readProperties(): Properties
+}
```

Comment:
I don't quite get the abstraction here. Can you elaborate on this interface a bit?

I like, that you created an interface for property reading. However, I'm not quite sure, if this interface is the right abstraction. The only implementation of this interface is the `VersionPropertiesReader` which reads properties from the predetermined version file.
Therefore, you're basically abstracting away how the version file is read, instead of abstracting away how properties files are read. I think, that latter might be a better abstraction.

Something like this as a guidance:

~~~~Kotlin
interface PropertiesReader {
  fun readProperties(from: String): Properties
}

class ResourcesPropertiesReader: PropertiesReader {
}
~~~~

Reply:
You're right, we might have other properties files, which should be read from different paths, I just didn't think about them yet.